Add synchronise section in readme (#137)
OperationalBina/PipeRT2: README.md

  } 
}
```

# Synchroniser

> In the pipe there is synchronising mechanism which used to synchronise the routines FPS.

Reviewer: ```suggestion
In the pipe there is a synchronising mechanism which is used to synchronise the routine's FPS.
```

> This mechanism force routines to rest if their FPS significantly higher than the bottlenecks routines.

Reviewer: ```suggestion
This mechanism forces routines to rest, if their FPS is significantly higher than that of the bottlenecks routines.
```

> It saves resources and should not affect the number of the processed routines. 

Reviewer: ```suggestion
It saves resources, and should not affect the number of the processed routines. 
```

> The common use for using the synchronising mechanism is the case there are fast routines, 
> and their next routines in the flow have lower FPS routines.

Reviewer: ```suggestion
The best example of a case where the synchronising mechanism would be useful, is when there are fast routines
followed by routines with lower FPS.
```
